### PR TITLE
Post comment edit delete add feat

### PR DESCRIPTION
**Pull request by samyakshah3008**
1. Create a Post
I can create a post with a "+" button or "Create new post" button which will open a textbox and will have a "Post" or "Submit" button.

2. Edit & Delete Post and comment

- Edit Post and comment
I can click on the "Edit Post" button which will open the same component with the textbox where we can edit a post & save it.

- Delete Post and comment
I can click on the "Delete Post" button which will delete a post.

### CODE REVIEW

File: src/components/Comments.jsx
Comment (Developers-codz) [inline src/components/Comments.jsx:25]: Appreciation -  for using Chakra UI

File: src/components/Comments.jsx
Comment (Developers-codz) [inline src/components/Comments.jsx:28]: importing react can be skipped as its no longer needed after version 17 also remove empty line

File: src/components/UserPost.jsx
Comment (Developers-codz) [inline src/components/UserPost.jsx:110]: remove console if no longer needed

File: src/components/UserPost.jsx
Comment (Developers-codz) [inline src/components/UserPost.jsx:115]: here too

File: src/components/UserPost.jsx
Comment (Developers-codz) [inline src/components/UserPost.jsx]: remove commented code

File: src/store/postSlice.js
Comment (Developers-codz) [inline src/store/postSlice.js:123]: here

File: src/store/postSlice.js
Comment (Developers-codz) [inline src/store/postSlice.js:164]: here

File: src/store/postSlice.js
Comment (Developers-codz) [inline src/store/postSlice.js:218]: suggestion -  handle rejected and pending states as well

**Review by Developers-codz — APPROVED** [review]
Good code quality and folder structure.
Added a few suggestions else its good to go 💯 💯

File: src/store/postSlice.js
Comment (samyakshah3008) [reply src/store/postSlice.js:218]: Thank you for detailed review, Kajal!

**Comment by samyakshah3008** [comment]
Looks good to merge after one approval.

**Pull request merged**
